src/components/Table/TableContainer.jsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import Heading from 'components/Heading';
 import styles from './TableContainer.css';
 
-const TableContainer = ({ title, style, children }) => (
-  <div className={styles.container} style={{ ...style }}>
+const TableContainer = ({ title, style, children, className }) => (
+  <div className={className} style={{ ...style }}>
```

Comment:
This should have a default style, it shouldn't be required to pass in a className. That className should just override the default values.